api/src/services/script-runner.service.ts

```diff
@@ -134,6 +138,58 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param amiChartImportDTO this is a string it takes a very specific formatted like:
+   * percentOfAmiValue_1 householdSize_1_income_value householdSize_2_income_value \n percentOfAmiValue_2 householdSize_1_income_value householdSize_2_income_value
+   * @returns a stringified version of AmiChartCreate DTO
+   * @description transfers data from foreign data into the database this api normally connects to. From this you can use it
```

Comment:
Last sentence seems like it's maybe unfinished?

Reply:
oop! I updated a couple things about the jsdocs!